README.md

```diff
@@ -45,7 +45,25 @@ npm install
 npm start
 ```
 
-Then, browse to http://localhost:8080
+Then, browse to http://localhost:4001
+
+## Development - Docker
+
+Create a `.docker.env.dev` file containing the environment variables that you'd like to configure for running the app. You can see the `.docker.env.example` file for all the available environment variables, or server/config.js for defaults and descriptions.
+
+Build the NodeJS docker image. You only need to run this once. You will need to rebuild this only if `Dockerfile.dev` changes.
+
+```
+docker build -f Dockerfile.dev -t firefox-send .
+```
+
+Run the server:
+
+```
+docker run --env-file .docker.env.dev -v $PWD:/app -p 4001:4001 firefox-send
```

Comment:
I like to run with `-it` since it shows the npm crap actually happening in real time

also you get pretty colors

Reply:
Maybe we should make a script that does all this setup/run work for us

Reply:
> I like to run with -it since it shows the npm crap actually happening in real time

Well, that's just, like, your opinion, man.

> also you get pretty colors

I'm sold. Can you add a commit?

Reply:
> Maybe we should make a script that does all this setup/run work for us

Sure, but I don't really know Perl that well. I'll ask Jason if he'll take one for the team.

Reply:
Does it have to be only in Perl? I'd like to make sure we're using the best tool for the job. I propose Java 5

Reply:
Oh, good call. Thanks for looking out for us! But I don't want to install Java on my machine, so can you dockerize that too?

Reply:
k done

Reply:
Thanks you! And good call on adding that temporary run script to tide us over while Jason works on the proper version. :+1:

Reply:
Gonna merge this now actually! Jason is going to make a new PR for his script.